content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -107,46 +106,30 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-Note: in Istio release 1.7, we are introducing a new way of measuring performance by enabling `jitter` in
-the load generator. It helps by modeling random traffic from the client side when using connection pools. In the next
-section, we will present both `jitter` and `non-jitter` performance measurements.
-
 ### Latency for Istio {{< istio_release_name >}}
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
 proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
-the two proxies add about 2.65 ms and 2.91 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
-After enabling `jitter`, those numbers reduced to 1.7 ms and 2.69 ms, respectively. We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
+the two proxies add about 1.7 ms and 2.7 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
 for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
 
-{{< image width="90%"
-    link="latency_p90_fortio_without_jitter.svg"
-    alt="P90 latency vs client connections"
-    caption="P90 latency vs client connections without jitter"
->}}
-
-{{< image width="90%"
-    link="latency_p99_fortio_without_jitter.svg"
-    alt="P99 latency vs client connections"
-    caption="P99 latency vs client connections without jitter"
->}}
-
 {{< image width="90%"
     link="latency_p90_fortio_with_jitter.svg"
     alt="P90 latency vs client connections"
-    caption="P90 latency vs client connections with jitter"
+    caption="P90 latency vs client connections"
 >}}
 
 {{< image width="90%"
     link="latency_p99_fortio_with_jitter.svg"
     alt="P99 latency vs client connections"
-    caption="P99 latency vs client connections with jitter"
+    caption="P99 latency vs client connections"
```

Comment:
Does it remove all discussion about jitter?
We should still mention jitter here, so that if someone tries to run benchmarks with other tools, these numbers will look non reproducible.

Reply:
I did intentionally remove all discussion of jitter because it has been causing confusion (people thinking 'jitter' is an Istio feature, or that benchmarks without jitter are valid).

We do link to the exact scripts we run